code/modules/sanity/inspiration_component.dm

```diff
@@ -73,6 +62,17 @@
 	else
 		return get_stats.Invoke()
 
-/datum/component/inspiration/proc/get_power()
-	var/list/L = calculate_statistics()
-	return L[STAT_MEC]
+/datum/component/inspiration/proc/get_power() // used to determine how powerful the artifact is for the matterforge
+	var/power = 0
+	if(perk)
+		var/list/L = calculate_statistics()
```

Comment:
I don't see the use of var/list/L anywhere in this function.

Reply:
look below, it sets power